test/unittests/evm_calls_test.cpp

```diff
@@ -342,7 +341,11 @@ TEST_F(evm_calls, call_then_oog)
     accounts[call_dst] = {};
     call_result.status_code = EVMC_FAILURE;
     call_result.gas_left = 0;
-    execute(1000, "6040600060406000600060aa60fef180018001800150");
+
+    const auto code =
+        ::call(0xaa).gas(254).value(0).input(0, 0x40).output(0, 0x40) + 4 * add(OP_DUP1) + OP_POP;
```

Comment:
Here `call()` clashes with `.call()` from `MockedHost`. This can be solved by compositing (instead of inheriting) of `MockedHost`. This should also improve general readability by having `host.accounts[x]`, `host.call_result`, etc.

Reply:
This new syntax looks great!